src/services/encryption/data_encryption.ts

```diff
@@ -1,9 +1,9 @@
 "use strict";
 
-const crypto = require('crypto');
-const log = require('../log.js');
+import crypto = require('crypto');
+import log = require('../log');
 
-function arraysIdentical(a, b) {
+function arraysIdentical(a: Buffer, b: Buffer) {
```

Comment:
I suggest using `Array<any> | Buffer` as a data type for the arguments, to be more suited to the function name.